packages/worker/src/worker.ts

```diff
@@ -365,8 +365,8 @@ export class NativeWorker implements NativeWorkerLike {
   public readonly shutdown: Promisify<OmitFirstParam<typeof native.workerShutdown>>;
 
   public static async create(options: CompiledWorkerOptionsWithDefaults): Promise<NativeWorkerLike> {
-    const nativeWorker = await promisify(native.newWorker)({
-      ...options,
+    const core = await promisify(native.newCore)({
+      maxCachedWorkflows: 0,
```

Comment:
To confirm: This will be the shared core later on, right? And that's where the max cached numbers will get set to something?

Reply:
Yes, this will come in a later PR